packages/react/src/progress-indicator/ProgressIndicator.test.tsx

```diff
@@ -1,18 +1,68 @@
 import '@testing-library/jest-dom';
 import 'html-validate/jest';
 import { axe, toHaveNoViolations } from 'jest-axe';
-import { render, cleanup } from '../../../../test-utils';
+import { render, screen } from '../../../../test-utils';
 import { ProgressIndicator, ProgressIndicatorProps } from './ProgressIndicator';
 import {
-	ProgressIndicatorItem,
-	ProgressIndicatorItemStatus,
 	activeProgressStatusAllowedList,
 	allProgressStatuses,
-} from './ProgressIndicatorItem';
+	ProgressIndicatorItemStatus,
+	ProgressIndicatorItem,
+} from './utils';
 
 expect.extend(toHaveNoViolations);
 
-afterEach(cleanup);
```

Comment:
[cleanup is done automatically nowadays](https://kentcdodds.com/blog/common-mistakes-with-react-testing-library#using-cleanup)